packages/flutter/lib/src/painting/image_cache.dart

```diff
@@ -5,6 +5,7 @@
 import 'image_stream.dart';
 
 const int _kDefaultSize = 1000;
+const int _kDefaultSizeBytes = 10000000; // 10 MB
```

Comment:
10485760 ?

Reply:
Wouldn't that be a MiB?

Reply:
I mostly am saying we should use a power of 2. I find the whole "Mibibyte" naming idea to be crazy but I'm not going to die on that hill. :-)